jib/README.md

```diff
@@ -149,3 +138,16 @@ spec:
     - name: IMAGE
       value: gcr.io/my-project/my-app
 ```
+
+Run the build:
+
+```shell
+kubectl apply -f jib-gradle-build.yaml
+```
+
+If you would like to customize the container, configure the `jib-gradle-plugin` in your `pom.xml`. 
+See [setup instructions for Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#setup) for more information.
+
+### Speed up builds
+
+See [Speed up builds](#speed-up-builds).
```

Comment:
Even though it's annoying to have two copies, it might be helpful to be explicit about the steps to speed up Gradle builds specifically. I'm thinking if I'm a heavy Gradle user and doesn't know anything about Maven, I don't want to have to figure out which parts of the Maven-specific "speeding up builds" doc apply to me and which don't.

Let me know if that's a silly thought, and that Gradle developers would necessarily know enough about Maven to figure it out, I don't know the Javas.